crates/ty_python_semantic/resources/mdtest/public_types.md

```diff
@@ -0,0 +1,421 @@
+# Public types
+
+## Basic
+
+The "public type" of a symbol refers to the type that is inferred for a symbol from an enclosing
```

Comment:
```suggestion
The "public type" of a symbol refers to the type that is inferred for a symbol from an external (either enclosing or nested)
```

Reply:
> I don't think we ever look at public types of symbols of symbols in nested scopes

Don't we? This reveals `int`, not `Literal[1]`:

```py
class Foo:
    X: int = 1

reveal_type(Foo.X)
```

Or am I confusing the concept of "public types" with something else?